PacketDotNet/IPProtocolType.cs

```diff
@@ -14,8 +14,6 @@ namespace PacketDotNet
     [SuppressMessage("ReSharper", "InconsistentNaming")]
     public enum IPProtocolType : byte
     {
-        /// <summary> Dummy protocol for TCP. </summary>
-        IP = 0,
```

Comment:
Please keep, this is ideal as fallback for parse issues but can probably be named as INVALID.

Reply:
Ah, yes, I missed that, this is OK then.